privacy-snapshot.json

```diff
@@ -52,6 +52,7 @@
   "static.cx.metamask.io",
   "swap.api.cx.metamask.io",
   "test.metamask-phishing.io",
+  "thumb.canalplus.pro",
```

Comment:
I believe this is unintentional

Reply:
Yup removing; i think it happened when running e2e test locally with ---update-privacy-snapshot flag

Reply:
Removed [14a7051](https://github.com/MetaMask/metamask-extension/pull/27629/commits/14a70515c6dfc0ecc3ff589ff347c51644df2048)